### PR TITLE
set global font

### PR DESCRIPTION
**Pull request by amorgulis**
Set `Roboto Condensed` as the primary font in the theme so it doesn't need to be set on an individual component basis; Fixed a piece of UI that should've been using `Oswald` font.

### CODE REVIEW

File: src/common/constants.tsx [hunk 1]
```diff
@@ -8,18 +8,14 @@ export const theme = createMuiTheme({
...
-      'sans-serif',
-      'Roboto Condensed'
-    ].join(','),
+    fontFamily: ['Roboto Condensed', 'Oswald', 'sans-serif'].join(','),
   },
 });
 export interface DrawerControlProps {
-   isOpen: boolean, 
-   onClose: () => void 
+  isOpen: boolean;
```
Comment (Kai-ros) [inline src/common/constants.tsx:15]: NIT: What's the stylistic difference of semi-colon over comma here?

File: src/components/CaseCard/CaseCard.tsx [hunk 2]
```diff
@@ -90,7 +89,11 @@ const CaseCard = (props: CaseProps) => {
               <ListItem>
                 <ListItemText
                   onClick={onPOIClick}
-                  primary={primaryText(props.case.CaseNumber)}
+                  primary={
+                    <span className={classes.primaryText}>
```
Comment (Kai-ros) [inline src/components/CaseCard/CaseCard.tsx:93]: I mentioned this in another PR, should we not be opting for <div> when possible over <span> ?

File: src/components/Header/CaseHeader.tsx [hunk 3]
```diff
@@ -1,77 +1,78 @@
...
+  headerItem: {
+    color: '#ffffff',
+    fontSize: '12px',
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    letterSpacing: '0.05em',
+  },
 }));
 
 interface IPoiType {
```
Comment (Kai-ros) [inline src/components/Header/CaseHeader.tsx:34]: Can we change this to POIProps or something similar? This ISomethingSomething is a naming convention Sam injected I don't understand..

File: src/components/Header/CaseHeader.tsx [hunk 4]
```diff
@@ -1,77 +1,78 @@
...
+            <Grid item key={index}>
+              <Button
+                autoFocus={index === 0}
+                className={classes.button}
+                size="large"
+                onClick={() => {
+                  setPoi(poi);
+                }}
+              >
+                <span className={classes.headerItem}>{poi}</span>
```
Comment (Kai-ros) [inline src/components/Header/CaseHeader.tsx:67]: Same as above, `<div>` instead of `<span>`

File: src/components/Header/CaseSubHeader.tsx [hunk 5]
```diff
@@ -1,91 +1,91 @@
...
+    marginTop: '10px',
+    lineHeight: '16px',
+    fontWeight: 'normal',
+    letterSpacing: '0.002em',
+  },
 }));
 
-interface IPoiInput{
-    poi: string;
+interface IPoiInput {
```
Comment (Kai-ros) [inline src/components/Header/CaseSubHeader.tsx:44]: Same as above, update to POIProps or something similar.

File: src/components/POI/PoiCard.tsx [hunk 6]
```diff
@@ -1,105 +1,117 @@
...
+  moreIcon: {
+    color: '#ffffff',
+    marginRight: '4vh',
+  },
+  divider: {
+    background: '#c3c7c5',
+  },
 }));
 
 interface IPoiCard {
```
Comment (Kai-ros) [inline src/components/POI/PoiCard.tsx:72]: Update name.

File: src/common/constants.tsx (hunk 1 above)
Comment (amorgulis) [reply src/common/constants.tsx:15]: That got changed from `,` to `;` automatically by `prettier`. There's no functional difference, it's just a style thing. Apparently `prettier` deems `;` to be the "correct" way ;-)

File: src/components/CaseCard/CaseCard.tsx (hunk 2 above)
Comment (amorgulis) [reply src/components/CaseCard/CaseCard.tsx:93]: We just need an element to apply some CSS to the text so there's no real reason it needs to be a block level (`div`). Inline level (`span`) accomplishes that.

File: src/components/Header/CaseHeader.tsx (hunk 3 above)
Comment (amorgulis) [reply src/components/Header/CaseHeader.tsx]: Changed to `CaseHeaderProps`.

File: src/components/Header/CaseSubHeader.tsx (hunk 5 above)
Comment (amorgulis) [reply src/components/Header/CaseSubHeader.tsx]: Fixed.

File: src/components/POI/PoiCard.tsx (hunk 6 above)
Comment (amorgulis) [reply src/components/POI/PoiCard.tsx]: Done.

File: src/common/constants.tsx (hunk 1 above)
Comment (Kai-ros) [reply src/common/constants.tsx:15]: Lol Cool

**Review by Kai-ros — APPROVED** [review]

**Pull request merged**